my_list.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*args)
+    @list = args
+  end
```

Comment:
- [ ] The #initialize method that accepts multiple arguments. Please 🙏🏽 refactor your code thanks to that, calling the constructor with a list of numbers as in the test code is possible `MyList.new(1, 2, 3, 4)`

Reply:
Hello @franclobo, kindly i would like to say The splat operator (*) is used in the parameter list of the initialize method to indicate that it should accept an arbitrary number of arguments. When the method is called with arguments, they are automatically packed into an array called args, which can be used inside the method.